composer.json

```diff
@@ -1,31 +1,25 @@
-{
-    "name": "LibreEHR",
-    "type": "project",
-    "description": "LibreEHR - Free Open Source Electronic Health Records based on OpenEMR",
-    "keywords": ["libreehr", "openemr"],
-    "authors": [
-        {
-            "name": "Ken Chapple",
-            "email": "ken@mi-squared.com"
-        }
-    ],
-    "require": {
-        "php": ">=5.3.0"
+{ 
+    "name" : "LibreEHR",
+    "type" : "project",
+    "description" : "LibreHealth EHR - Free Open Source Electronic Health Records",
+    "keywords" : [ "libreehr", "openemr" ],
+    "authors" : [ { 
+        "name" : "Ken Chapple",
+        "email" : "ken@mi-squared.com"
+    } ],
+    "require" : { 
+        "php" : ">=5.3.0"
```

Comment:
Will our stuff run on 5.3?  I am almost certain that it must be 5.4, and possibly 7.0 specifically.